toolkit/tools/imagegen/configuration/kernelcommandline.go

```diff
@@ -49,7 +49,7 @@ func (k *KernelCommandLine) IsValid() (err error) {
 
 	// A character needs to be set aside for use as the sed delimiter, make sure it isn't included in the provided string
 	if strings.Contains(k.ExtraCommandLine, k.GetSedDelimeter()) {
-		return fmt.Errorf("ExtraCommandLine contains character %s which is reserved for use by sed", k.GetSedDelimeter())
```

Comment:
I t hink these should match the actual field names in the types. We can put quotes around them maybe?

Reply:
Ah, good catch! I'll fix that.